### PR TITLE
added parachain id validation on server

### LINKED ISSUE #213: Validate parachain ID
We are not validating the ID of a parachain, just sending to that chain when we get a user request.

We need to verify if such id is correct, maybe by having a list or a configuration?

### PR DESCRIPTION
**Pull request by Bullrich**
Simple check to verify that the parachain id is a number and it is between 1000 and 9999.

Closes #213

### CODE REVIEW

File: src/faucet.e2e.ts [hunk 1]
```diff
@@ -150,4 +160,24 @@ describe("Faucet E2E", () => {
...
+    } catch (error) {
+      if (axios.isAxiosError(error)) {
+        const axiosError = error as AxiosError<{ error: string }>;
+        expect(axiosError.response?.data.error).toEqual(
+          "Parachain invalid. Be sure to set a value between 1000 and 9999",
+        );
+      } else {
+        throw error;
+      }
+    }
```
Comment (mutantcornholio) [inline src/faucet.e2e.ts:181]: Wouldn't it work somewhat like this?

```ts
const promise = webEndpoint.post("/drip/web", { address: userAddress, recaptcha: "anything goes", parachain_id: "100" });

await expect(promise).rejects.toThrow(AxiosError);
await expect(promise).rejects.toMatchObject({response: {data: {
  error: "Parachain invalid. Be sure to set a value between 1000 and 9999"
}}});
```

Comment (Bullrich) [reply src/faucet.e2e.ts:181]: It would! Thank you for that!

Comment (mordamax) [reply src/faucet.e2e.ts:181]: `throw new Error("Unexpected success");` lmao

**Review by mordamax — APPROVED** [review]

File: src/faucet.e2e.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const randomAddress = () => createTestKeyring().addFromSeed(randomAsU8a(32)).add
 describe("Faucet E2E", () => {
   const matrix = axios.create({ baseURL: "http://localhost:8008" });
   const webEndpoint = axios.create({ baseURL: "http://localhost:5555" });
-  const PARACHAIN_ID = 100; // From the zombienet config.
+  const PARACHAIN_ID = 1000; // From the zombienet config.
```
Comment (rzadp) [inline src/faucet.e2e.ts:14]: So we have to change the zombienet config as well - it's [here](https://github.com/paritytech/polkadot-testnet-faucet/blob/570eb9e731bb32f9cb60e0dabf8595d3118ec57a/e2e/zombienet.native.toml#L17).

Comment (Bullrich) [reply src/faucet.e2e.ts:14]: Is this config from zombie net valid, or was it set only for the e2e tests?

Comment (Bullrich) [reply src/faucet.e2e.ts:14]: IT PASSED!

**Review by mutantcornholio — APPROVED** [review]

**Pull request merged**